src/main/java/hudson/scm/IntegrityCheckpointAction.java

```diff
@@ -327,7 +346,11 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
       listener.getLogger().println(sqlex.getMessage());
       LOGGER.log(Level.SEVERE, "SQLException", sqlex);
       return false;
-    }
+    } catch (Exception e) {
+      LOGGER.severe("Exception caught...");

```

Comment:
print exception then `LOGGER.log(Level.SEVERE, "...", e)`. Otherwise this system log message is pretty useless to admins